src/index.ts

```diff
@@ -140,7 +166,8 @@ export async function gru<MasterArgs = undefined>(
 
     // If # of workers is set to 0, no workers will be started
     // And just run worker start function in master process
-    if (options.workers === 0) {
+    const useDedicatedWorkers: boolean = (!!options.dedicatedWorkers && Object.keys(options.dedicatedWorkers).length > 0)
+    if (options.workers === 0 && !useDedicatedWorkers) {
```

Comment:
Unsure about this change. If `useDedicatedWorkers` is true then the dedicated workers will start, but no normal workers will?

Reply:
Either remove `&& !useDedicatedWorkers` or start all dedicated workers inline?